lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -385,7 +385,6 @@ namespace Js
 
         // Construct HSTRING of timeZoneId passed
         IfFailThrowHr(GetWindowsGlobalizationLibrary(scriptContext)->WindowsCreateStringReference(timeZoneId, static_cast<UINT32>(wcslen(timeZoneId)), &timeZoneHeader, &timeZone));
-        Assert(timeZone);
```

Comment:
Alright, I went ahead and checked and seems all other asserts are valid i.e. if they are hit, there is a problem.